Source/Extensions/DictionaryExtensions.swift

```diff
@@ -52,6 +51,34 @@ public extension Dictionary {
 		return jsonData?.string(encoding: .utf8)
 	}
     
+    /// SwifterSwift: Count dictionary entries that where function returns true.
+    ///
+    /// - Parameter where: condition to evaluate each tuple entry against.
+    /// - Returns: Count of entries that matches the where clousure.
+    public func count(where condition: @escaping ((key: Key, value: Value)) throws -> Bool ) rethrows -> Int {
+        var count : Int = 0
+        try self.forEach {
+            if try condition($0) {
+                count += 1
+            }
+        }
+        return count
+    }
+    
+    /// SwifterSwift: Return every key of the dictionary as a tuple array filtered by a clousure. There's no order guaranteed.
+    ///
+    /// - Parameter where: condition to evaluate each tuple entry against.
+    /// - Returns: Filtered key/value of dictionary as a tuple array.
+    public func tuples(where condition: @escaping ((key: Key, value: Value)) throws -> Bool) rethrows ->  [(Key, Value)] {
+        return try self.flatMap { try condition($0) ? $0 : nil }
+    }
+    
+    /// SwifterSwift: Return every key of the dictionary as a tuple array. There's no order guaranteed.
+    ///
+    /// - Returns: key/value of dictionary as a tuple array.
+    public func tuples() -> [(Key,Value)] {
```

Comment:
I feel like this extension is not a good fit for SwifterSwift. 
It specializes the generic abstraction created by `map` -- It is just `map { $0 }`